media/css/firefox/nothing-personal/styles.scss

```diff
@@ -12,92 +12,236 @@ $image-path: '/media/protocol/img';
+}
+
+@media #{$mq-lg} {
+    // non-grid layout, under main section
+    .c-aside {
+        display: flex;
+        justify-content: center;
+        margin-top: $spacing-xl;
+        text-align: center;
+    }
+
+    .c-aside-link {
+        color: $color-white;
         display: flex;
         flex-direction: column;
         align-items: center;
-        overflow-x: hidden;
+        padding: $spacing-sm;
+        text-decoration: none;
+        cursor: default;
+
+        + * {
+            margin-left: $spacing-md;
+        }
+
+        &:hover,
+        &:active,
+        &:focus {
+            color: $color-white;
+        }
+
+        &:active,
+        &:focus {
+            background-color: $color-blue-60;
+        }
     }
+}
 
-    aside {
-        display: none;
+.c-text-section {
+    margin-bottom: $spacing-xl;
+
+    * {
+        color: $color-white; // IE fallback
+        color: var(--body-text-color-inverse);
     }
+}
 
-    @media (prefers-reduced-motion: no-preference) and (#{$mq-sm}) {
-        background-attachment: fixed;
+@supports (display: grid) {
+    .c-grid {
+        display: grid;
+        gap: $spacing-md;
+
+        &.mzp-l-content::after {
+            display: none; // remove pseudo-element spacing
+        }
+
+        @media #{$mq-md} {
+            grid-template-columns: repeat(2, minmax(0, 1fr));
+
+            &.mzp-t-content-md {
+                max-width: $screen-md;
+            }
+        }
     }
 
+    // sidebar layout
     @media #{$mq-lg} {
-        aside {
-            display: flex;
+        .c-content-layout {
+            display: grid;
+            grid-template-columns: 1fr [main-start] auto [main-end aside-start] 1fr [aside-end];
+        }
+
+        .c-main {
+            grid-area: main;
+        }
+
+        .c-aside {
+            grid-area: aside;
+            justify-content: unset;
+            margin-top: unset;
+            justify-self: start;
+            margin-left: -#{$layout-lg}; // shift over padding
             flex-direction: column;
-            position: absolute;
-            text-align: center;
-            top: 105px;
-            right: 0;
-
-            a, .c-trash {
-                color: $color-white;
-                display: flex;
-                flex-direction: column;
-                align-items: center;
-                margin-bottom: $spacing-2xl;
-                padding: $spacing-sm;
-                text-decoration: none;
-                cursor: default;
-            }
+        }
 
-            a {
-                &:hover,
-                &:active,
-                &:focus {
-                    color: $color-white;
-                }
+        .c-aside-link {
+            margin-bottom: $spacing-2xl;
+            margin-left: unset;
+        }
+    }
+}
 
-                &:active,
-                &:focus {
-                    background-color: $color-blue-60;
-                }
-            }
+.c-text-title {
+    @include font-size(34px);
```

Comment:
I think this is the Mozilla font-size scale on Firefox themed page, so the custom property vars were slightly off from Figma in ways that seemed worth overriding for this case